radis/lbl/calc.py

```diff
@@ -541,7 +543,8 @@ def _check_molecules_are_consistent(
             mole_fraction, diluent, molecule
         )
         if 'molecule' in kwargs_molecule:
-            del kwargs_molecule['molecule']
+          del kwargs_molecule['molecule']
+        engine = config["DATAFRAME_ENGINE"]
```

Comment:
@1someshverma i'm seeing this change here : in your next changes, you can remove this line, remove the `engine` argument from `_calc_spectrum_one_molecule`, and read engine directly from `_calc_spectrum_one_molecule`.  No need to have it here already

@menasrac you can ignore this comment !

Reply:
@1someshverma resolve the comment when implemented in your branch